extern/tbb/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+# Silence FetchContent warnings with CMake >=3.24.
+if( POLICY CMP0135 )
+   cmake_policy( SET CMP0135 NEW )
+endif()
+
+# Tell the user what's happening.
+message( STATUS "Building TBB as part of the TRACCC project" )
+
+# Declare where to get TBB from.
+set( TRACCC_TBB_SOURCE
+   "URL;https://github.com/oneapi-src/oneTBB/archive/refs/tags/v2021.7.0.tar.gz;URL_MD5;68e617448f71df02d8688c84d53778f6"
```

Comment:
Why 2021.7.0 and not 2021.7.1?

Reply:
Where do you see 2021.7.1? :confused:
  - https://github.com/oneapi-src/oneTBB/releases
  - https://github.com/oneapi-src/oneTBB/tags

I thought I picked up the latest "blessed" version... Though even if I didn't, that shouldn't matter for our project all that much.